NYPL/scsb_item_updater
# TGR-109 Update Ruby version + GHA

## JIRA tickets:
- [SCSB Item Updater - update Ruby version + GHA](https://newyorkpubliclibrary.atlassian.net/browse/TGR-109)

## Things Done:
- Updated OS / Ruby / Rails / Gems
- Implemented GitHub Actions for PR builds
- Implemented GitHub Actions for deployments
- Set `qa` as the default branch and added branch protection rules to `qa` and `production`

## How to Test:

- Build the app locally via README instructions.
- Run the unit tests
- Check out the services running in the qa environment (nypl-digital-dev)
